docs/app/pages/charts/sections/spark-charts/spark-charts/spark-charts.component.html

```diff
@@ -115,6 +118,95 @@
     </table>
 </div>
 
+<p>UX Aspects offers a wide range of chart colors. Set the color of the chart by using one of the following classes:</p>
```

Comment:
Since the angular 4 component allows arbitrary colors via barColor and trackColor, this description needs to be a bit different. Also the color service for Angular 4 has not yet been documented. I think doing this will take quite a bit more effort, so for now can you just revert the changes in the Angular 4 component? Sorry for the backtrack.